Droidcon-Boston/app/src/test/java/com/mentalmachines/droidcon_boston/data/ScheduleTest.kt

```diff
@@ -1,8 +1,7 @@
 package com.mentalmachines.droidcon_boston.data
 
 import com.mentalmachines.droidcon_boston.data.Schedule.ScheduleRow
-import org.junit.Assert.assertFalse
-import org.junit.Assert.assertTrue
+import org.junit.Assert.*
```

Comment:
try to avoid wildcard imports here too...Detekt will flag them

Reply:
Did this get merged? It's still showing as open to me.

Reply:
CI/CD PR was merged last week: https://github.com/Droidcon-Boston/conference-app-android/pull/139